molecule/oauth-kafka-connect-replicator-mtls-rhel/prepare.yml

```diff
@@ -5,6 +5,11 @@
 - name: Provision Oauth Server
   import_playbook: ../oauth.yml
 
+- name: Download OAuth IDP cert
+  hosts: localhost
+  tasks:
+    - shell: openssl s_client -showcerts -connect localhost:8443 </dev/null 2>/dev/null|openssl x509 -outform PEM > oauthcertfile.pem
```

Comment:
Since all the prepare files for https enabled oauth server have this task should we take it out and put in a separate file to make it easy in case we make some changes (like say we change the port on which keycloak server runs https). 
And similarly also for downloading sso certfile.

Reply:
creating a file for just one task is an overkill, we can create a file if more such tasks are needed for oauth/idp in future